indico/modules/events/contributions/util.py

```diff
@@ -160,10 +161,13 @@ def render_contrib_report(self, contrib=None):
         total_entries = contrib_report_kwargs.pop('total_entries')
         tpl_contrib = get_template_module('events/contributions/management/_contribution_report.html')
         tpl_reports = get_template_module('events/management/_reports.html')
-        fragment = tpl_reports.render_displayed_entries_fragment(len(contrib_report_kwargs['contribs']), total_entries)
+        contribs = contrib_report_kwargs['contribs']
+        total_duration = (sum([c.duration for c in contribs], timedelta()),
```

Comment:
Initially I had it there but I had to remove it. I'll try to remember the reason